packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -34,18 +40,16 @@ const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = (config
             fontSize: toPx(typeRamp.t8.vp3.fontSize),
             lineHeight: toPx(typeRamp.t8.vp3.lineHeight),
             paddingBottom: toPx(7),
-            ...setFloatAndClear(),
-            "& + div": {
-                marginTop: "0",
-                ...setFloatAndClear(),
-                "& + span": {
-                    ...setFloatAndClear(false),
-                    [applyLocalizedProperty("margin-left", "margin-right", config.direction)]: toPx(5),
-                }
-            }
+            ...setFloatAndClear()
         },
         toggle_wrapper: {
-            position: "relative"
+            position: "relative",
+            marginTop: "0",
+            ...setFloatAndClear()
+        },
+        toggle_statusLabel: {
+            ...setFloatAndClear(false),
+            [applyLocalizedProperty("margin-left", "margin-right", config.direction)]: toPx(5)

```

Comment:
I think we are mixing and matching our tasks here, I would think this PR would only be about adding the defaults for the design system but it appears to include toggle fixes as well, can we address this in another PR or is there a reason they need to be included in this one?

Reply:
The styling will be broken without passing a class here. Why - I don't know, but I can remove. We just need to accept that there will be a gap in coverage for default styles for toggle. With a class, things pass as expected; with previous changes, it fails to pass the styles.

Reply:
So, by not assigning a class here, the task is complete but broken for toggle. I'm fine with that if need be.